### ISSUE TITLE
Locked containers are unable to be broken, causing compatibility issues

### ISSUE THREAD

**Issue by The834th** [elBukkit/MagicPlugin#1095]
If containers like chests and dispensers are locked, magic outright prevents them from being broken to avoid their items dropping on the ground, simply sending a message to the player:
![image](https://user-images.githubusercontent.com/23151186/182507198-859a0877-51af-4e6d-865b-0c0fb4b92d24.png)
In some scenarios, this could be helpful, but it causes compatibility issues with any pack or plugin that uses locked containers as custom blocks, as they can't be broken.
Unlocking these containers with lockpick before breaking them works, but that adds another step to something that should be very simple.
Is there a way to disable this function?

**Comment by NathanWolf** [comment]
Yes, you can do `/mconfig config protect_locked false` to turn this feature off.
